frontend/src/views/Combat.vue

```diff
@@ -343,6 +329,54 @@ export default {
     setFightMultiplier() {
       localStorage.setItem('fightMultiplier', this.fightMultiplier.toString());
     },
+
+    setStaminaSelectorValues(){
```

Comment:
this needs to be done way better, there is too much repetitive code here. I will assist.

Reply:
Alright I was thinking about something like having an array with all options and then just removign the ones who dont pass

Reply:
take a look at my solution :3

Reply:
Nice solution! I was trying something simmilar just the other way arround I guess
But Im still trying to figure out how to avoid the if else wall :P
![image](https://user-images.githubusercontent.com/87876521/127676937-b9d3d377-0c21-4d9c-8da4-ea4669b9262b.png)

Reply:
there are other ways, like doing a filter on an array, but I was not feelin it